src/interfaces/query-irrelevant-results.ts

```diff
@@ -0,0 +1,9 @@
+export class QueryIrrResult {
```

Comment:
irr->irrelevantEntities

Reply:
should it be in utils?

Reply:
it will be used in each driver (db) cause its a wrapper type generated when querying for irrelevant entities